docs/assets/css/main.css

```diff
@@ -10457,7 +10458,9 @@ button.bg-dark:focus {
 }
 
 .text-center {
-    text-align: center!important
+    text-align: center!important;
+    justify-content: center;
+    margin-left: 550px;
```

Comment:
I haven’t had a chance to build this latest update, yet. I wonder if this static margin centers the footer content irrespective of browser width. @raghavaggarwal99 in your tests, has the firm maintained center when the browser is resized (from mobile to desktop width)?